test/AutoBus.spec.ts

```diff
@@ -0,0 +1,104 @@
+import AutoBus from '../src/AutoBus';
+
+describe('AutoBus', () => {
+  describe('#join', () => {
+    it('throws if no roomname', (done) => {
+      const bus = new AutoBus();
+      try {
+        // @ts-expect-error
```

Comment:
Fix so that we don't have `ts-expect-error`. Everything should work without errors.

Reply:
I suppose in this case it's okay, since we're testing a JS consumption of the method. 🤔